juju/python-libjuju
# [JUJU-3525] Add juju 3.1.2 missing facades.

#### Description

Add missing facades included in juju 3.1.2

#### QA Steps

Bootstrap a 3.1.2 controller, connect using pytyhon-libjuju and check that all the facades are known.
